server/app.js

```diff
@@ -53,6 +54,22 @@ app.get(
   productHandlers.getProductsByIDHandler
 );
 
+/******* */
+//Clients
+//****** */
+
+app.get(
+  buildAPIPath("/clients/:clientID"),
+  clientHandlers.getClientByIDValidatorChain,
```

Comment:
after this validatorchain there should be the parameter referencing the middleware (handling bad requests). you should see it in other route methods